test/test-readability.js

```diff
@@ -279,7 +279,7 @@ describe("Test pages", function() {
 
       runTestsWithItems("jsdom", function(source) {
         var doc = new JSDOM(source, {
-          url: uri,
+          url: uri
```

Comment:
Nit: we prefer trailing commas for multiline object declarations. Again, I should really sort out or eslint setup...

Reply:
Fixed!